engine/build.gradle

```diff
@@ -231,24 +238,33 @@ task createVersionInfoFile {
     onlyIf { env.BUILD_URL != null }
     doLast {
         versionInfoFileDir.mkdirs()
-        ant.propertyfile (file: versionInfoFile) {
-            ant.entry(key:'buildNumber',value:env.BUILD_NUMBER)
-            ant.entry(key:'buildId',value:env.BUILD_ID)
-            ant.entry(key:'buildTag',value:env.BUILD_TAG)
-            ant.entry(key:'buildUrl',value:env.BUILD_URL)
-            ant.entry(key:'jobName',value:env.JOB_NAME)
-            ant.entry(key:'gitBranch',value:convertGitBranch(env.GIT_BRANCH))
-            ant.entry(key:'gitCommit',value:env.GIT_COMMIT)
-            ant.entry(key:'dateTime',value:startDateTimeString)
-            ant.entry(key:'displayVersion',value:displayVersion)
-            ant.entry(key:'engineVersion',value:version)
+        ant.propertyfile(file: versionInfoFile) {
+            ant.entry(key: 'buildNumber', value: env.BUILD_NUMBER)
+            ant.entry(key: 'buildId', value: env.BUILD_ID)
+            ant.entry(key: 'buildTag', value: env.BUILD_TAG)
+            ant.entry(key: 'buildUrl', value: env.BUILD_URL)
+            ant.entry(key: 'jobName', value: env.JOB_NAME)
+            ant.entry(key: 'gitBranch', value: convertGitBranch(env.GIT_BRANCH))
+            ant.entry(key: 'gitCommit', value: env.GIT_COMMIT)
+            ant.entry(key: 'dateTime', value: startDateTimeString)
+            ant.entry(key: 'displayVersion', value: displayVersion)
+            ant.entry(key: 'engineVersion', value: version)
         }
     }
 }
 
+
+//TODO: Remove it  when gestalt will can to handle ProtectionDomain without classes (Resources)
+task copyResourcesToClasses(type:Copy) {
+    from sourceSets.main.output.resourcesDir
+    into sourceSets.main.output.classesDirs.first()
+}
+
+jar.dependsOn copyResourcesToClasses
+
 jar.dependsOn createVersionInfoFile
 jar.dependsOn cacheReflections
-jar.finalizedBy cleanReflections
+//jar.finalizedBy cleanReflections
```

Comment:
is this still required? if not, please remove, else it shouldn't be commented out